lib/ed25519-browser.js

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as ed25519 from '@stablelib/ed25519';
+import {randomBytes} from '@stablelib/random';
+
+export default {
+  // use stablelib's randomBytes in the browser
+  randomBytes,
+  ...ed25519
```

Comment:
Instead of exposing `randomBytes` here, let's do:

```suggestion
  ...ed25519,
  generateKeyPair
```

And define `generateKeyPair` to call `ed25519.generateKeyPair` and use `randomBytes` internally.

Reply:
Addressed this here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/11d6eb9cf0ce47777e6fe4ec4e6386a922b94ca5